dev-db/postgrest-bin/postgrest-bin-0.4.4.0.ebuild

```diff
@@ -0,0 +1,69 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit user eutils systemd
+
+MY_PN="${PN/-bin/}"
+
+DESCRIPTION="PostgREST serves a fully RESTful API from existing PostgreSQL database (Binary)"
+HOMEPAGE="https://postgrest.com"
+
+uri() {
```

Comment:
what's the purpose of that? I see upstream provides normal targzipped file, w/o arch-dependent things